Sources/TuistKit/Services/TestService.swift

```diff
@@ -65,6 +65,7 @@ final class TestService {
         self.simulatorController = simulatorController
         self.contentHasher = contentHasher
         self.cacheDirectoryProviderFactory = cacheDirectoryProviderFactory
+        try? ProcessEnv.setVar(Constants.EnvironmentVariables.tuistBuildForDevelopment, value: "true")
```

Comment:
- Can't we pass this when invoking xcodebuild instead of blindly setting it in the init?
- if for some reason we can't, can we at least remove it from the init (e.g. move to the run below)